docs/source/faq.rst

```diff
@@ -494,3 +494,33 @@ There are two kinds of constrained optimizations, one with soft constraints and
 Soft constraints do not have to be satisfied, but an objective function is penalized if they are unsatisfied. On the other hand, hard constraints must be satisfied.
 
 Optuna is adopting the soft one and **DOES NOT** support the hard one. In other words, Optuna **DOES NOT** have built-in samplers for the hard constraints.
+
+How can I parallelize optimization
```

Comment:
Add `?`.

Reply:
Thank you for pointing this out. The correction has been addressed.